index.html

```diff
@@ -28,6 +28,87 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span id="headline-gree
         </ul>
     </section>
     </main>
+    <section class="works">
```

Comment:
- [ ] Please make sure this container of has a `display:grid` property as stated in the requirement😎

Reply:
Done